modules/tinymce/src/core/main/ts/api/PublicApi.ts

```diff
@@ -0,0 +1,170 @@
+/**
```

Comment:
This is the file used to define the public API. It's then compiled and used by the new rollup plugin to bundle the final .d.ts file.

Reply:
If anyone can think of something else that should be exposed here, let me know.